R/plot.check_normality.R

```diff
@@ -10,6 +10,9 @@
 #' @param colors Character vector of length two, indicating the colors (in
 #'   hex-format) for points and line.
 #' @param detrend Logical that decides if the plot should be detrended.
+#' @param method The method used for estimating the qq/pp bands. Default to
+#'   `"ell"` (equal local levels / simultaneous testing - recommended). See
+#'   `qqplotr::stat_qq_band()` for more options and details.
```

Comment:
I think we should mention the other options here as well, so users don't have to click through several help-pages.

Reply:
That would require we keep track of the options `qqplotr` offers. Honestly, best if users don't touch this :P

Reply:
The argument in `qqplotr::stat_qq_band()` is differently named, so it's hard to follow / find out, which options to use. Therefore, I'd still suggest adding valid options here to the docs.

Reply:
Done